### PR TITLE
Update release instructions for automation

### PR DESCRIPTION
**Pull request by jtuttle**
Releases are not automatically created when a tag is pushed. This commit updates
the release instructions to take this into account.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,28 +84,20 @@ The following checklist should be followed when creating a release:
...
+
+- [ ] Publish the draft as "pre-released".
+
 - [ ] Update homebrew tools
-  - [ ] In [`cyberark/homebrew-tools`](https://github.com/cyberark/homebrew-tools) repo, update
-        the [`summon.rb` formula](https://github.com/cyberark/homebrew-tools/blob/main/summon.rb#L4-L6) with a PR
-        using the file `dist/goreleaser/summon.rb`.
-        
+  - [ ] In [`cyberark/homebrew-tools`](https://github.com/cyberark/homebrew-tools)
+        repo, create a PR to update the [`summon.rb` formula](https://github.com/cyberark/homebrew-tools/blob/main/summon.rb#L4-L6)
```
Comment (diverdane) [inline CONTRIBUTING.md:96]: Did you mean to highlight only lines 4-6, and not the entire file?

**Review by diverdane — APPROVED** [review]
LGTM!
Approved, but had a quick (curiosity, mainly) question on the highlighting of lines 4-6 in the `summon.rb` file.

**Review by imheresamir — APPROVED** [review]
LGTM!

File: CONTRIBUTING.md (hunk 1 above)
Comment (jtuttle) [reply CONTRIBUTING.md:96]: Not sure if that was intended, those lines were already highlighted in the previous version. I'll just leave it for now.

**Pull request merged**